### ISSUE TITLE
Running "dxf auth" against a registry without any authentication service should not fail

### ISSUE THREAD

**Issue by mrueg** [davedoesdev/dxf#17]
Due to 
```
        if response.status_code != requests.codes.unauthorized:
            raise exceptions.DXFUnexpectedStatusCodeError(response.status_code,
                                                          requests.codes.unauthorized)
```
`dxf auth` returns a `dxf.exceptions.DXFUnexpectedStatusCodeError: expected status code 401, got 200.`
It might be worth to return an empty bearer token instead or handle that case differently (as the request was HTTP200 successfull).

**Comment by davedoesdev** [comment]
Interesting - I thought the unit tests do no auth. I'll take a look soon.

**Comment by davedoesdev** [comment]
Ah, it's testing with `htpasswd`. You're right, it needs to be tested with no auth too.

**Comment by davedoesdev** [comment]
Actually, it should be tested without auth, strange, wonder what the tests do in that case...

**Comment by davedoesdev** [comment]
Fixed in 7.1.0. Again, thanks for the heads up, appreciate it.

**Comment by mrueg** [comment]
Thanks for the quick fix! 👍
